attorch/regularizers.py

```diff
@@ -14,10 +13,8 @@ class Laplace(nn.Module):
     """
     def __init__(self):
         super().__init__()
-
         self.conv = nn.Conv2d(1, 1, 3, bias=False, padding=1)
         self.conv.weight.data.copy_(torch.from_numpy(laplace()))
-        self.conv.weight.requires_grad = False
```

Comment:
That's really weird. You are right, it should be there, but in my code, it read `self.conv.weight.requires_grad = False`, which should be there.